app/src/main/java/campus/tech/kakao/contacts/RecyclerViewAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package campus.tech.kakao.contacts
+
+import android.content.Context
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.content.ContextCompat.startActivity
+import androidx.recyclerview.widget.RecyclerView
+
+class RecyclerViewAdapter(
+	val contact_list: List<Contact>,
+	val inflater: LayoutInflater,
+	val context: Context
+) : RecyclerView.Adapter<RecyclerViewAdapter.ViewHolder>(){
+
+	inner class ViewHolder(itemView: View):RecyclerView.ViewHolder(itemView){
+		val name_first: TextView
+		val name:TextView
+		init {
+			name_first = itemView.findViewById(R.id.nameFirst)
+			name = itemView.findViewById(R.id.name)
+			//클릭 리스너 생성
+			itemView.setOnClickListener {
+				val position = adapterPosition
+				//adapterPosition을 통해 포지션을 받음.
+				val contact = contact_list.get(position)
+//				Toast.makeText(context, ""+contact.name+" "+contact.phone, Toast.LENGTH_SHORT).show()
```

Comment:
불필요한 코드 주석은 삭제해주세요. 꼭 필요하다면 왜 주석을 했는지 이유도 함께 밝혀주셔야 합니다. 예를 들어
```suggestion
// TODO: 개발 이후 삭제 예정
//				Toast.makeText(context, ""+contact.name+" "+contact.phone, Toast.LENGTH_SHORT).show()
```
과 같은 식으로여! 물론 이 예시의 경우 그냥 삭제하는게 맞습니다